### DISCUSSION TITLE
How to use backticks and take a screenshot

### DISCUSSION THREAD

**Discussion by TheChronicMonster** [reach-sh/reach-lang#1251] [General]
### Backticks
Please use backticks or screenshots when sharing information from your machine. 

Wrap single lines of code in single "`" backticks. 
(The backtick is the key to the left of "1").

`Single backtick example`

Wrap multiple lines in three (3) backticks. "```"

```
Multiple
Line
Example
```

### Screenshots
Please do not submit cell phone photos of your screen. It's much easier to troubleshoot with a clear screenshot.

See this helpful site for how to take a screenshot on your machine: https://www.take-a-screenshot.org/

**Comment by jeapostrophe** [comment]
It is even better to upload the entire file to http://gist.github.com --- You can use that site to upload multiple files, including the output of running `reach` commands. If you don't feel comfortable sharing the entire program publicly, feel free to directly DM someone on the team (like me!) and I'll look at your code secretly :)
